scripts/integration-test.sh

```diff
@@ -14,5 +14,3 @@ if [[ $RET == 99 ]]; then
 elif [[ $RET != 0 ]]; then
     exit $RET
 fi
```

Comment:
Are the integration tests still being run in this scenario?

Reply:
What scenario? The changes here, aside from decoupling linting, are from #738